recipes/sos-notebook/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91fc849a9d8c0b5fb975ec82eb1f2b9be2603ff936b8241a928cc8d738a57e16
+
+build:
+  skip: True  # [win]
```

Comment:
I think this can be removed now as [`ptyprocess` is now `noarch`](https://anaconda.org/conda-forge/ptyprocess/files):
```suggestion
```

Reply:
Indeed. Maybe they overheard our discussion. :-)

@jdblischak Could you try to remove this line and see if the tests pass?